README.md

```diff
@@ -166,6 +168,7 @@
 - ["How to create a Github Profile README with Dynamic Github Stats"](https://codewithghazi.com/how-to-create-a-github-profile-readme-with-dynamic-github-stats/) - *Ghazi Khan*
 - ["How I Built A Self-Updating README On My Github Profile"](https://www.mokkapps.de/blog/how-i-built-a-self-updating-readme-on-my-git-hub-profile/) - *Michael Hoffmann (Mokkapps)*
 - ["Building a self-updating profile README for GitHub"](https://simonwillison.net/2020/Jul/10/self-updating-profile-readme/) - *Simon Willison*
+- [Markdown Badges](https://dev.to/ileriayo/mardown-badges-2og0)
```

Comment:
Provide some description

Reply:
A getting started article on markdown badges.

Reply:
I mean provide description in the README

Reply:
Done.